xml/src/main/java/org/ehcache/xml/ConfigurationParser.java

```diff
@@ -105,6 +112,23 @@
   private final Map<URI, CacheResourceConfigurationParser> resourceXmlParsers = new HashMap<URI, CacheResourceConfigurationParser>();
   private final ConfigType config;
 
+  static String getReplacementString(String originalValue) {
+    Matcher matcher = SYSPROP.matcher(originalValue);
+
+    StringBuffer sb = new StringBuffer();
+    while (matcher.find()) {
+      final String property = matcher.group(1);
+      final String value = System.getProperty(property);
+      if (value == null) {
+        return null;
```

Comment:
Well I think that's confusing though. When you see `/home/asnaps/ehcache${missing.prop}` you directly know what's going on.